website/blog/2021-01/01/_index.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Happy anniversary Gardener! Three years of open source Kubernetes management
+linkTitle: Happy anniversary Gardener! Three years of open source Kubernetes management
```

Comment:
```suggestion
linkTitle: Happy anniversary Gardener!
```
`linkTitle` is limited to 2-3 short words. This is the main title that appears in the news feed on the front page and is limited in space